api/lib/domain/usecases/handle-badge-acquisition.js

```diff
@@ -0,0 +1,52 @@
+const _ = require('lodash');
+
+const handleBadgeAcquisition = async function ({
+  assessment,
+  domainTransaction,
+  badgeCriteriaService,
+  badgeAcquisitionRepository,
+  badgeRepository,
+  knowledgeElementRepository,
+  targetProfileRepository,
+}) {
+  if (assessment.campaignParticipationId !== null) {
```

Comment:
Je sais que ce n'est pas le propos de la PR, mais ça me paraît être un BSR intéressant. Les comparaisons de ce type à null manquent, à mon sens, d'expressivité.
Je me demande si cette ligne ne peut pas être remplacée par :
```suggestion
  if (assessment.isForCampaign()) {
```